spec/models/ncr/work_order_spec.rb

```diff
@@ -1,4 +1,6 @@
 describe Ncr::WorkOrder do
+  include ProposalSpecHelper
```

Comment:
That's just `require`ing the file...I'm only `include`ing the mixin in certain places.